src/services/fireblocks.service.js

```diff
@@ -253,11 +253,13 @@ class FireblocksClient {
         && e.response.data
         && e.response.data.code === TX_ALREADY_EXISTS_ERROR_CODE
       )) {
+        // eslint-disable-next-line no-console
```

Comment:
It would be good, if possible

Reply:
Since the wallet can only start 1 fireblocks wallet per instance I thought that the fireblocks specific logs would not create a doubt on which wallet it comes from.
But I guess it would be nicer to be more consistent with this logger, so I changed to include the logger instance